nginx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# a minimal Nginx container including containerbuddy and a simple virtualhost config
```

Comment:
Then I'd keep objects that are at the same level of abstraction at the same level of directory. I think you're missing that the Cloudflare app doesn't actually work without an example app behind it.